GitHubCodespaces.ps1

```diff
@@ -0,0 +1,538 @@
+            $OwnerName = $elements.ownerName
+            $RepositoryName = $elements.repositoryName
+
+            $telemetryProperties['OwnerName'] = Get-PiiSafeString -PlainText $OwnerName
+            $telemetryProperties['RepositoryName'] = Get-PiiSafeString -PlainText $RepositoryName
+
+            $uriFragment = "repos/$OwnerName/$RepositoryName/codespaces"
+            $description = "Getting $OwnerName/$RepositoryName/codespaces"
+
+            break
+        }
+    }
+
+    $params = @{
+        'UriFragment' = $uriFragment
+        'Description' = $description
+        'AcceptHeader' = "$script:nebulaAcceptHeader,$script:baptisteAcceptHeader,$script:mercyAcceptHeader"
+        'AccessToken' = $AccessToken
+        'TelemetryEventName' = $MyInvocation.MyCommand.Name
+        'TelemetryProperties' = $telemetryProperties
+    }
+
+    $result = Invoke-GHRestMethodMultipleResult @params
+    if ($result.codespaces) {
+        $result = $result.codespaces
+    }
+
+    return ($result | Add-GitHubCodespaceAdditionalProperties)
+}
+
+function Start-GitHubCodespace
+{
+<#
+    .SYNOPSIS
+        Start a user's codespace.
+
+    .DESCRIPTION
+        You must authenticate using an access token with the codespace scope to use this endpoint.
+
+        GitHub Apps must have write access to the codespaces_lifecycle_admin repository permission to use this endpoint.
+
+        The Git repo for this module can be found here: http://aka.ms/PowerShellForGitHub
+
+    .PARAMETER CodespaceName
+        The name of the codespace.
+
+    .PARAMETER PassThru
+        Returns the updated GitHub Issue.  By default, this cmdlet does not generate any output.
+        You can use "Set-GitHubConfiguration -DefaultPassThru" to control the default behavior
+        of this switch.
+
+    .PARAMETER Wait
+        If present will wait for the codespace to start.
+
+    .PARAMETER AccessToken
+        If provided, this will be used as the AccessToken for authentication with the
+        REST Api.  Otherwise, will attempt to use the configured value or will run unauthenticated.
+
+    .INPUTS
+        GitHub.Codespace
+
+    .OUTPUTS
+        GitHub.Codespace
+
+    .EXAMPLE
+        Start-GitHubCodespace -Name vercellone-effective-goggles-qrv997q6j9929jx8
+
+    .LINK
+        https://docs.github.com/en/rest/codespaces/codespaces?apiVersion=2022-11-28#start-a-codespace-for-the-authenticated-user
+#>
+    [CmdletBinding(
+        SupportsShouldProcess,
+        ConfirmImpact='Low')]
+    [OutputType({$script:GitHubCodespaceTypeName})]
+    [Diagnostics.CodeAnalysis.SuppressMessageAttribute("PSReviewUnusedParameter", "", Justification="PassThru is accessed indirectly via Resolve-ParameterWithDefaultConfigurationValue")]
+    param(
+        [Parameter(Mandatory,
+            ValueFromPipeline,
+            ValueFromPipelineByPropertyName)]
+        [string] $CodespaceName,
+
+        [switch] $PassThru,
+
+        [switch] $Wait,
```

Comment:
The preference is that "standard" switches like `PassThru` or `Force` (along with `AccessToken`) are the last parameters in a function call.  Please fix the order here and in the docs above.